gravitee-am-service/src/main/java/io/gravitee/am/service/reporter/builder/management/VerifyAttemptAuditBuilder.java

```diff
@@ -0,0 +1,48 @@
+/**
+ * Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.gravitee.am.service.reporter.builder.management;
+
+import io.gravitee.am.common.audit.EntityType;
+import io.gravitee.am.common.audit.EventType;
+import io.gravitee.am.model.User;
+import io.gravitee.am.model.VerifyAttempt;
+
+/**
+ * @author Ashraful Hasan (ashraful.hasan at graviteesource.com)
+ * @author GraviteeSource Team
+ */
+
+public class VerifyAttemptAuditBuilder extends ManagementAuditBuilder<VerifyAttemptAuditBuilder> {
+    public VerifyAttemptAuditBuilder() {
+        super();
+    }
+
+    public VerifyAttemptAuditBuilder verifyAttempt(VerifyAttempt verifyAttempt, User user) {
```

Comment:
open discussion:

Maybe we should create a dedicated method to assign the user to the target & actor ?
```

public VerifyAttemptAuditBuilder verifyAttempt(VerifyAttempt verifyAttempt)
public VerifyAttemptAuditBuilder user(User user)
```

Reply:
make sense, done